packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.test.tsx

```diff
@@ -287,4 +287,29 @@ describe('DetailsList', () => {
     }, 0);
     jest.runOnlyPendingTimers();
   });
+
+  it('invokes optional onColumnResize callback per IColumn if defined when columns are adjusted', () => {
+    const detailsList = mount(
+      <DetailsList
+        items={mockData(2)}
+        skipViewportMeasures={true}
+        // tslint:disable-next-line:jsx-no-lambda
+        onShouldVirtualize={() => false}
+      />
+    );
+
+    const columns: IColumn[] = mockData(2, true);
+    columns[0].onColumnResize = jest.fn();
+    columns[1].onColumnResize = jest.fn();
+
+    // componentWillReceiveProps not executed on initial render in test
+    // so we need to force one via setProps and update.
+    const newProps = { columns };
+
+    detailsList.setProps(newProps);
+    detailsList.update();
```

Comment:
I was able to get the unit test working by using the above... cc: @natalieethell